module/documents/actor.mjs

```diff
@@ -29,6 +29,13 @@ export default class ActorEd extends Actor {
     return this.items.filter( item => item.type === 'namegiver' )[0];
   }
 
+  /**
```

Comment:
I'd strongly recommend to use @inheritdoc here, since we don't change the functionality of the superclass function, and we'd have to adapt it if the super function changes..
I think you can still add the userfunction tag additionally.

Reply:
just like this?

 /**
   * @inheritDoc  
   * @description        Pre-process a creation operation of the actor document to ensure that a prototype token is
   * @UserFunction      UF_TokenResouces-preCreate
   */